rachellegelden/Home-Together-Canada
# HTC -343: Tooltip

# [Issue 343](https://github.com/rachellegelden/Home-Together-Canada/issues/343)

closes #343 

## Summary
- Installed `react-tooltip` in client 
- Created new common components for `Checkbox`, `LargeTextArea`, `YNButton`
- Created a new `ToolTip` component with its own test and CSS
- Added Tooltip to Business & Member Registration
- Wrote custom information for each tooltip on each page
- Added ToolTip to `signInInfo` component 
- Updated all affected snapshots
- Added 
`jest.mock("react-tooltip/node_modules/uuid", () => ({ v4: () => "00000000-0000-0000-0000-000000000000" }));`
to tests that include tooltip due to its dynamic UUID. By hardcoding its UUID, we are able to test and the components that have tooltip.


## Relevant Motivation & Context
- Help users with filling out the form 

## Testing Instructions
- Go to member / business form, view each tooltip and ensure the wording is accurate 

## Developer checklist prior to opening this pull request:


## Reviewer
